### ISSUE TITLE
Adding a new practice when your author/perspective is not merged will fail the build

### ISSUE THREAD

**Issue by springdo** [openpracticelibrary/openpracticelibrary#204]
**What is this about**
The CMS workflow will create a new author on a separate branch so if you in parallel create a new practice, the build will fail for this until the author is merged because a page _has_ to be associated with a given primary author.

**The solution**
1. better way to add new author to avoid this race condition
or
2. better error handling/  guard in the Hugo template.

**Comment by rdebeasi** [comment]
Interesting fact: author IDs are case-sensitive, so uppercase letters in the author name will trigger this problem (e.g., #226). If this issue is fixed, uppercase letters in the author name won't break the build, but they won't associate with the correct author either.

I wonder if it would be worth going back to the dropdown UI for this, which has its own set of tradeoffs (i.e., can't create a new practice and a new author in the same PR with the CMS). If so, let's make a new issue for that change.
